includes/class-wc-stripe-order-handler.php

```diff
@@ -333,6 +334,50 @@ public function cancel_payment( $order_id ) {
 			do_action( 'woocommerce_stripe_process_manual_cancel', $order );
 		}
 	}
+
+	/**
+	 * Filter. Adds additional meta data to Tracks events.
+	 * Note that this filter is only called if WC_Site_Tracking::is_tracking_enabled.
+	 *
+	 * @since 4.5.1
+	 * @param array Properties to be appended to.
+	 * @param string Event name, e.g. orders_edit_status_change.
+	 */
+	public function woocommerce_tracks_event_properties( $properties, $prefixed_event_name ) {
+		// Not the desired event? Bail.
+		if ( 'wcadmin_orders_edit_status_change' != $prefixed_event_name ) {
+			return $properties;
+		}
+
+		// Properties not an array? Bail.
+		if ( ! is_array( $properties ) ) {
+			return $properties;
+		}
+
+		// No payment_method in properties? Bail.
+		if ( ! array_key_exists( 'payment_method', $properties ) ) {
+			return $properties;
+		}
+
+		// Not stripe? Bail.
+		if ( 'stripe' != $properties[ 'payment_method' ] ) {
```

Comment:
Do we care about other Stripe payment methods? I tested with `stripe_multibanco` orders and the metadata wasn't tracked in those cases. I assume this would also happen for other alternatives, such as SEPA or iDeal.

Reply:
@bmccotter - each Stripe payment method ("stripe" (debit/credit cards), giro, multibanco, sofort, etc) comes across as separate tracks events. Right now I am ONLY adding the metadata to the debit/credit cards "stripe" one. Should we add this meta data to all the Stripe methods?

Reply:
@allendav no need here, as this is US only so other payment methods don't yet apply.